systemtest/src/main/java/io/strimzi/systemtest/AbstractST.java

```diff
@@ -214,7 +215,7 @@ void replaceMirrorMakerResource(String resourceName, Consumer<KafkaMirrorMaker>
         replaceCrdResource(KafkaMirrorMaker.class, KafkaMirrorMakerList.class, DoneableKafkaMirrorMaker.class, resourceName, editor);
     }
 
-    void replaceBridgeResource(String resourceName, Consumer<KafkaBridge> editor) {
+    protected void replaceBridgeResource(String resourceName, Consumer<KafkaBridge> editor) {
```

Comment:
Is this change needed ? :)

Reply:
Not sure for 100% not, but when I implemented it, there were problem that bridge tests are "in different package" so we can't use private methods there (from `AbstractST`). I will double check after other reviews and make changes if it's not needed.